docs/draft.md

```diff
@@ -0,0 +1,69 @@
+## Pull the Percona XtraBackup image
+
+`Docker pull` retrieves a Docker image from a registry and downloads the image to the local machine. To pull a specific version of a Docker image, add a tag with the version number.
+
+The following command retrieves the `8.0.34` version:
+
+```{.bash data-prompt="$"}
+$ docker pull percona/percona-xtrabackup:8.0.34
+```
+
+List the downloaded Docker images:
+
+```{.bash data-prompt="$"}
+$ docker images
+```
+
+??? example "Expected output"
+
+    ```{.text .no-copy}
+    REPOSITORY                   TAG       IMAGE ID       CREATED        SIZE
+    percona/percona-xtrabackup   8.0.34    a10a980cae0a   3 months ago   579MB
+    ```
+
+## Connect to a Percona Server for MySQL container
+
+Percona XtraBackup works in combination with a database server. When running a Docker container for Percona XtraBackup, you can make backups for a database server installed in a separate Docker container.
+
+To set up Percona Server for MySQL in a Docker container, follow the instructions in [Quickstart Percona Server for MySQL with Docker](https://docs.percona.com/percona-server/8.0/quickstart-docker.html).
+
+Run Percona Server for MySQL container:
+
+```{.bash data-prompt="$"}
+$ docker run -d --name=psmysql -e MYSQL_ROOT_PASSWORD=secret --volume=myvol:/var/lib/mysql percona/percona-server:8.0.34
+```
+
+As soon as Percona Server for MySQL runs and you add some data to it, you are ready to make backups with Percona XtraBackup.
+
+When running Percona XtraBackup from a container and connecting to a MySQL server container, we recommend using the `–-user root` option in the Docker command.
+
+## Run a container from Percona XtraBackup image
+
+Create a Docker container based on Percona XtraBackup image with the docker run command. If it is not the first time you use the selected image, Docker uses the image available locally.
+
+1. Create docker network
+
+    ```{.bash data-prompt="$"}
+    $ docker network create mynet
+    ```
+
+2. Create a Docker container based on Percona XtraBackup image with the docker run command. 
```

Comment:
Should we add that tag to the container so that it matches the PS version? Otherwise, in a year, this quickstart may not work.